tests/acceptance/helpers/webdavHelper.js

```diff
@@ -192,44 +192,30 @@ exports.createFolder = function (user, folderName) {
  * @param {string} user
  * @param {string} fileName
  * @param {string} contents
- * @param {number} waitMaxIfExisting
  */
-exports.createFile = async function (user, fileName, contents = '', waitMaxIfExisting = 10000) {
+exports.createFile = async function (user, fileName, contents = '') {
+  const davPath = exports.createDavPath(user, fileName)
   /**
    * makes sure upload operations are carried out maximum once a second to avoid version issues
    * see https://github.com/owncloud/core/issues/23151
    */
-  uploadTimeStamps[user] = uploadTimeStamps[user] || {}
-
-  const pollCheck = async (retries = 0, waitFor = 100, waitMax = waitMaxIfExisting) => {
-    if (!uploadTimeStamps[user][fileName] || waitMax <= waitFor) {
-      return
-    } else {
-      uploadTimeStamps[user][fileName] = true
+  if (uploadTimeStamps[user] && uploadTimeStamps[user][fileName]) {
+    const timeSinceLastFileUpload = Date.now() - uploadTimeStamps[user][fileName]
+    if (timeSinceLastFileUpload <= 1600) {
+      await client.pause(1600 - timeSinceLastFileUpload)
```

Comment:
Adjusting this pause time to `1600` fixes the versions issue, the tests are passing even in multiple runs https://drone.owncloud.com/owncloud/web/20534/9/19